server/data/restClient.ts

```diff
@@ -133,6 +112,36 @@ export default class RestClient {
     return this.config.url
   }
 
+  private async postOrPut(
+    method: 'post' | 'put',
+    { path = '', headers = {}, responseType = '', data = {}, raw = false }: PutRequest | PostRequest = {},
+  ): Promise<unknown> {
+    logger.info(`${method} using user credentials: calling ${this.name}: ${path}`)
+    try {
+      const request =
+        method === 'post' ? superagent.post(`${this.apiUrl()}${path}`) : superagent.put(`${this.apiUrl()}${path}`)
+
+      const result = await request
+        .send(data)
+        .agent(this.agent)
+        .use(restClientMetricsMiddleware)
+        .retry(2, (err, res) => {
```

Comment:
Unrelated to these changes so feel free to ignore but thought you might like to know we removed this retry for PUT/POST. 
If a non-GET request fails there may be a validation error so it was causing headaches retrying